AppCreationScripts/apps.json

```diff
@@ -0,0 +1,33 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an Android app",
+    "Level": 200,
+    "Client": "Android"
+  },
+  "AppRegistrations": [
+    {
+      "x-ms-id": "active-directory-android-java",
```

Comment:
@rpdome do you think this should be "ms-identity-android-java"?

Reply:
I'm not familiar with AppCreationScripts, so I'm not sure what this is for. If it's just a name (and changing it wouldn't break anything), then I could change it to ms-identity-android-java.

Reply:
@archieag what is this value for?